ci_framework/roles/registry_deploy/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Install Podman package
+  become: true
+  package:
+    name: podman
+    state: present
+
+- name: Deploy the local registry
+  block:
+    - name: 'Deploy the local registry from {{ cifmw_rp_registry_image }}'
+      ansible.builtin.shell: >-
+        podman run
+        -d
+        --network=host
+        --name registry
+        -e REGISTRY_HTTP_ADDR={{ cifmw_rp_registry_ip }}:{{ cifmw_rp_registry_port }}
+        -e REGISTRY_LOG_LEVEL=debug
+        {{ cifmw_rp_registry_image }}
+      tags:
+        - skip_ansible_lint
+  rescue:
+    - name: "Deploy the local registry from {{ cifmw_rp_registry_image_fallback }}"
+      ansible.builtin.shell: >-
+        podman run
+        -d
+        --network=host
+        --name registry
+        -e REGISTRY_HTTP_ADDR={{ cifmw_rp_registry_ip }}:{{ cifmw_rp_registry_port }}
+        -e REGISTRY_LOG_LEVEL=debug
+        {{ cifmw_rp_registry_image_fallback }}
+      tags:
+        - skip_ansible_lint
+
+- name: Open port for content provider registry
+  become: true
+  ansible.builtin.command: >-
+        iptables -I INPUT -p tcp --dport {{ cifmw_rp_registry_port }} -j ACCEPT
```

Comment:
`iptables` per se is dead, long live nftables - all of tripleo (and, by extension, podified) is using `nft` CLI.